core/base-service/examples.js

```diff
@@ -154,7 +155,9 @@ function transformExample(inExample, index, ServiceClass) {
       style: style === 'flat' ? undefined : style,
       namedLogo,
     },
-    keywords,
+    keywords: keywords.concat(
+      categories.find(c => c.id === ServiceClass.category).keywords
+    ),
```

Comment:
Is it possible for `keywords` to ever be falsy here? Or do we initialize this as an empty array (if there's no explicitly defined keywords in the service def) beforehand?

Reply:
The Joi schema specifies `default([])`, so I _think_ it should be okay.

Reply:
Makes sense, everything looks good in the review app too 👍